domain/src/main/kotlin/no/nav/su/se/bakover/domain/oppdrag/simulering/Simulering.kt

```diff
@@ -17,12 +17,22 @@ data class Simulering(
 
     fun harFeilutbetalinger() = TolketSimulering(this).simulertePerioder.any { it.harFeilutbetalinger() }
 
+    /**
+     * Nettobeløpet påvirkes av skatt, så tas ikke med i equals-sjekken.
+     * Bruttobeløpet, altså summen av månedsbeløpene, brukes i stedet .
+     */
     override fun equals(other: Any?) = other is Simulering &&
         other.gjelderId == this.gjelderId &&
         other.gjelderNavn == this.gjelderNavn &&
-        other.nettoBeløp == this.nettoBeløp &&
         other.periodeList == this.periodeList &&
         other.bruttoYtelse() == this.bruttoYtelse()
+
+    override fun hashCode(): Int {
+        var result = gjelderId.hashCode()
+        result = 31 * result + gjelderNavn.hashCode()
+        result = 31 * result + periodeList.hashCode()
```

Comment:
Hvorfor hoppe over bruttoytelse?

Reply:
Dette var den IntelliJ genererte. Tenkte også på om bruttoytelse burde vært med, men det er jo en funksjon som man kanskje ikke ønsker å kjøre hver gang hashCode kjører 🤷 

Den oppfyller uansett funksjonen til hashCode, siden to like objekter vil få samme hashCode (det vi mister er noe teoretisk effektivitet siden den nå vil putte to ellers like objekter med forskjellig bruttobeløp i samme bøtte).